RenameAnnotationRector using hasByName() on null

# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | v0.7.26              |
| Installed as   | composer dependency                     |

rector fails with message: *[ERROR] Could not process "tests/SomeTest.php" file by "Rector\Renaming\Rector\Annotation\RenameAnnotationRector", due to:                                                                                                                     "Call to a member function hasByName() on null".*

## Minimal PHP Code Causing Issue

```php
<?php
// file tests/SomeTest.php
namespace tests;

class SomeTest extends \PHPUnit\Framework\TestCase
{
    /*
     * This is a comment and not a docblock
     */
    public function testSomething()
    {
        $this->assertTrue(false, 'Expected failure!');
    }
}
```

running: `rector process --set phpunit70 --dry-run tests/SomeTest.php`

The error is not thrown if instead of a comment enclosed by `/* .. */` is replaced by `/** ... */`.

The error is also present if `// ...` is used

## Expected Behaviour

Do not fail on `RenameAnnotationRector` rule.

## More information

Related to `ExceptionAnnotationRector`, a similar issue is present.
On commit #3199 issue #3179 is solved by checking if the instance returned by `$node->getAttribute(AttributeKey::PHP_DOC_INFO);` is not null, I think that the same check should be placed for `RenameAnnotationRector`.

IMO this is an easy PR. If you would like me to solve the issue by adding the null check after `$node->getAttribute()` I can make it, but I'm not familiar with rector development. Please advise.

PD: Thanks for rector and all this work!

--- a/rules/renaming/src/Rector/Annotation/RenameAnnotationRector.php
+++ b/rules/renaming/src/Rector/Annotation/RenameAnnotationRector.php
@@ -90,8 +90,11 @@ PHP
         /** @var Class_ $class */
         $class = $node->getAttribute(AttributeKey::CLASS_NODE);
 
-        /** @var PhpDocInfo $phpDocInfo */
+        /** @var PhpDocInfo|null $phpDocInfo */
         $phpDocInfo = $node->getAttribute(AttributeKey::PHP_DOC_INFO);
+        if ($phpDocInfo === null) {
+            return null;
+        }
 
         foreach ($this->classToAnnotationMap as $type => $annotationMap) {
             /** @var string $type */

--- a/rules/renaming/tests/Rector/Annotation/RenameAnnotationRector/Fixture/skip_method_with_comment_not_php_doc.php.inc
+++ b/rules/renaming/tests/Rector/Annotation/RenameAnnotationRector/Fixture/skip_method_with_comment_not_php_doc.php.inc
@@ -1,0 +1,18 @@
+<?php
+
+namespace Rector\Renaming\Tests\Rector\Annotation\RenameAnnotationRector\Fixture;
+
+/*
+ * This definition fails only when is extending \PHPUnit\Framework\TestCase
+ */
+final class FixtureRenameWithMethodWithCommentTest extends \PHPUnit\Framework\TestCase
+{
+    /*
+     * this is a comment, not a phpdoc
+     */
+    public function test()
+    {
+    }
+}
+
+?>

